### ISSUE TITLE
Falling through FP in Stormwind

### ISSUE THREAD

**Issue by MisterSitar** [SunwellTracker/issues#1770]
Decription: When arriving to Stormwind as a final destination for your FP upon landing you immediately fall through the ground and are located behind sewer gates

How it works: Take a flightpath to SW and there is a good 50/50 chance you'll fall through (have tried different clients in regards to regular one and edited one with Legion textures)

This occurs at least 2-3x a week.  Have personally never experienced it on another 3.3.5a server.

How it should work: You should not fall through the ground and/or the collision mesh should always be properly functioning.

![screenshot 203](https://user-images.githubusercontent.com/40922169/44313772-d61a1900-a3db-11e8-8621-3d1cce0407e7.png)

Source (you should point out proofs of your report, please give us some source):

**Comment by Akha72** [comment]
Yeah, I found out it happens mostly when I arrive in SW while alt-tabbing or having wow minimized.
